src/frontend/js/components/CourseProductCourseRuns/CourseRunSection.tsx

```diff
@@ -0,0 +1,31 @@
+import { PropsWithChildren } from 'react';
```

Comment:
This component doesn't seem to be tested, I didn't find the tests.

Reply:
This component is battle tested through component which are based on it `CourseRunList` and `EnrollableCourseRunList` and `EnrolledCourseRun`.

<img width="654" alt="image" src="https://user-images.githubusercontent.com/9265241/160653345-e566fede-daf6-4a8a-9af9-f93f74b80b66.png">

Reply:
I understand it is used in another component so through the other component it gets tested, but it's a bit confusing no, shouldn't a component be tested on its own, isn't it a convention. I thought so.

Reply:
I take your point it's confusing. But if I write tests for this component, I just test it twice ...

Reply:
yes but let's say your other component changes and doesn't use this one anymore, then the component itself won't be tested...don't know I would like another point of view on that, even If I'm happy with both, I personally add the case and tested it twice
@lunika